webapp/src/pages/offer/components/OfferComponent.js

```diff
@@ -0,0 +1,210 @@
+        super(props);
+  
+        this.state = {
+          userProfile : [],
+          loading: true,
+          error: "",
+          offer_id: 1,
+          rejected_reason:"",
+          rejected:false,
+          accepted: false,
+          offerList: [],
+          category:""
+        };
+      }
+    
+
+      handleChange = field => {
+        return event => {
+          this.setState({
+            [field.name]: event.target.value
+          });
+        };
+      };
+
+      buttonSubmit(){
+         const {offer_id,accepted,rejected,rejected_reason} = this.state;
+   
+        offerServices.updateOfferList(offer_id, DEFAULT_EVENT_ID, accepted, rejected, rejected_reason)
+        .then(response=>{
+          if (response.msg === "succeeded") {
+            this.getofferList();
+            this.setState({
+              addedSucess: true,
+              conflict: false,
+              notFound: false
+            });
+          } else if (response.msg === "404") {
+            this.setState({
+              addedSucess: false,
+              notFound: true,
+              conflict: false
+            });
+          } else if (response.msg === "409") {
+            this.setState({
+              notFound: false,
+              addedSucess: false,
+              conflict: true
+            });
+          }
+        })
+      }
+    
+    displayOfferContent = (e) => {
+      const {offerList,userProfile} = this.state;
+      console.log("<<< List  Function >>>>",offerList," userProfile ",userProfile)
+     return(   
+      <div className="container"  align="center" >
+      <p className="h5 pt-5">We are pleased to offer you a place at the Deep Learning Indaba 2019. Please see the details of this offer below </p>
+
+      <form class="pt-5 ">
+        <p class="font-weight-bold">You have been accepted as a  {userProfile!=null ? userProfile.user_category: "<Category>"} </p>
+        <p class="font-weight-bold">Your status</p>
+        <div class="row">
+          <div class="col-8 font-weight-bold">
+            Travel Award:
+          </div>
+          <div class="col-2">
+          {offerList!=null
+          ? offerList.travel_award
+          : "Not available"}
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col-8 font-weight-bold">
+            Accommodation Award:
+          </div>
+          <div class="col-2">
+          {offerList!=null
+          ? offerList.accomodation_award
+          : "Not available"}
+          </div>
+        </div>
+
+        <div class="row pb-5 ">
+          <div class="col-8 font-weight-bold">
+            Payment Required:
+          </div>
+          <div class="col-2">
+          {offerList!=null
+          ? offerList.payment_required
+          : "Not available"}
+          </div>
+        </div>
+        <p class="font-weight-bold">Please accept or reject this offer by {offerList!=null ? offerList.expiry_date : "<Expiry Date>"} </p>
```

Comment:
If there is no expiry date being pulled from backend, front-end should error, not just display "<Expiry Date>"